doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -60,6 +79,545 @@ Recommended changes
 
 The following changes are recommended for your application to work optimally after the migration.
 
+* For the Matter samples and applications
+
+  * The new API and helper modules have been added to the :file:`samples/matter/common` directory.
+    All Matter samples and applications have been changed to start using the common software modules.
+    You can follow the new approach and migrate your application to use the common software modules, what will signicantly reduce the size of a code required to be implemented in the application.
+    However you can also stay with the previous approach, but keep in mind that due to the structural differences, it may be harder to use Matter samples and aplications as a reference for your application using older convention.
```

Comment:
```suggestion
    You can also choose to keep using the previous approach, but due to the structural differences, it may be harder to use Matter samples and applications as a reference for an application using the older approach.
```